test/fixtures/upgrade/overlay/cainjector-ops.yaml

```diff
@@ -22,4 +22,4 @@ spec:
     spec:
       containers:
       #@overlay/match by=overlay.subset({"name": "cert-manager"})
-      - image: #@ "quay.io/jetstack/cert-manager-controller:{}".format(data.values.app_version)
+      - image: #@ "docker.io/library/cert-manager-cainjector-amd64:{}".format(data.values.app_version)
```

Comment:
note: this also updates the container name - @irbekrm agreed it probably should've been `cainjector` all along.